### PR TITLE
Add dummy for ACTION_BACKUP_SETTINGS

### PR DESCRIPTION
**Pull request by DaVinci9196**
Fix jump to backup and repair page crash exception. 
e.g.  Google Drive

### CODE REVIEW

File: play-services-core/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -756,6 +756,18 @@
             </intent-filter>
         </service>
 
+        <!-- backup -->
+
+        <activity
+            android:name="org.microg.gms.backup.component.BackupSettingsCollapsingActivity"
```
Comment (mar-v-in) [inline play-services-core/src/main/AndroidManifest.xml:762]: Can you rename to `org.microg.gms.backup.BackupSettingsActivity` please? There are not "component"s in backup and there's nothing "Collapsing" about that activity.

Comment (DaVinci9196) [reply play-services-core/src/main/AndroidManifest.xml]: > Can you rename to `org.microg.gms.backup.BackupSettingsActivity` please? There are not "component"s in backup and there's nothing "Collapsing" about that activity.

Okey

**Comment by Vavun** [comment]
Is this one broke translation repo ?

https://hosted.weblate.org/projects/microg/play-services-core-strings/#alerts

**Comment by mar-v-in** [comment]
Yeah, sometimes happens when you add new strings for translations. Weblate is working again now after manual sync.

**Pull request merged**